ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -15,10 +15,14 @@ let prototyped_of_field = function
       Some "22.26.0"
   | "VTPM", "persistence_backend" ->
       Some "22.26.0"
+  | "host", "recommended_guidances" ->
```

Comment:
Please remove the changes in this file out until it's time to merge to master, the changes are already outdated

Reply:
removed